datasets/meetbouten_azure/meetbouten/v2.0.0.json

```diff
@@ -0,0 +1,153 @@
+{
+  "id": "meetbouten",
+  "type": "table",
+  "version": "2.0.0",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "mainGeometry": "geometrie",
+    "identifier": "identificatie",
+    "required": [
+      "schema",
+      "identificatie"
+    ],
+    "display": "identificatie",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "identificatie": {
+        "type": "string",
+        "description": "Unieke identificatie van de meetbout"
+      },
+      "nabijAdres": {
+        "type": "string",
+        "description": "Beschrijving van een nabij adres"
+      },
+      "locatie": {
+        "type": "string",
+        "description": "Beschrijving van de locatie van de meetbout, bijvoorbeeld 'in gemeenschappelijke muur'"
+      },
+      "status": {
```

Comment:
Op dit moment kan het Amsterdam schema nog niet omgaan met `object` velden die geen relatie zijn. Deze zullen moeten worden "platgeslagen" tot `statusCode` en `statusOmschrijving`.
Op termijn is het wel de bedoeling om het gebruik van json blob velden te gaan implementeren.

Reply:
Zojuist `status` platgeslagen